### DISCUSSION TITLE
Detailed Example on how to use property graphs with Morph-KGC

### DISCUSSION THREAD

**Discussion by alloka** [morph-kgc/morph-kgc#296] [General]
Hello :)

My name is Ali and I'm a phd student in KU Leuven working with proerty graphs and knowledge graphs. I'm interested in how to use Morph-KGC on top of property graphs and I was looking for any detailed example or test-cases on that but couldn't find any. Can you please provide me with a detailed example on how to use Morph-KGC with property graphs?

Best regards,
Ali.

**Comment by arenas-guerrero-julian** [comment]
Hi @alloka ,

We have a [paper](https://oa.upm.es/83168/1/2024_iiWAS_PG_to_RML_star.pdf) related to this. You can check an example [here](https://github.com/arenas-guerrero-julian/pg2rml-star/tree/main/example) where you can find a [Kùzu](https://kuzudb.com/) database and the RML mapping to access it.
